### PR TITLE
Fix inspect linkages test

### PR DESCRIPTION
**Pull request by mingwandroid**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ build:
 test:
     commands:
         - test -f ${PREFIX}/lib/libuuid.a  # [unix]
-        - conda inspect linkages -n _test libuuid  # [unix]
+        - conda inspect linkages libuuid  # [unix]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Do you mind if we do?

```
    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
```

Comment (mingwandroid) [inline recipe/meta.yaml]: We skip the recipe on Windows anyway.

```
skip: True  # [win]
```

what does inspect objects do? I've never used it, inspect linkages works on macOS just fine btw.

Comment (ocefpaf) [inline recipe/meta.yaml]: It helps us to diagnose issues like: https://github.com/conda-forge/numpy-feedstock/issues/25 but I am not sure how important that is.

Comment (ocefpaf) [inline recipe/meta.yaml]: Regarding the `not win` syntax, it is not about Windows but rather my "pet peeve" with the `unix` selector, both `Linux` and `OS X` are not `Unix` :smile: 

With that said. I don't really care to change the selector, it was just in the lines I copied. I only care about the `objects` if you think they are important.

**Comment by ocefpaf** [comment]
@mingwandroid the Travs-CI failure was just a download issue and no new package is created here b/c it is just a test fix. Merging this to get you moving, but let me know your thoughts on the `conda inspect objects`.

**Comment by mingwandroid** [comment]
Go for it. Long term, once everyone's using conda-build 3, these things both need to go away and be done automatically at the end of every build, along with a security/hardening check.

**Comment by ocefpaf** [comment]
> these things both need to go away and be done automatically at the end of every build, along with a security/hardening check.

:heart_eyes:

**Pull request merged**